app/enterprise/1.3-x/deployment/access-license.md

```diff
@@ -24,13 +24,39 @@ Click into the **Files** section
 
 Click any file you would like to download.
 
-Alternatively, you can run this command in your terminal
+## Programmatically accessing your license file
 
+For programmatic access you'll need 3 pieces of information:
+
+ - username (was provided to by email by your CSE)
+ - repository name (the Bintray repository, visible in the GUI, usually named after your company name)
+ - the Bintray API key for the account (see below)
+
+To get the API key follow these steps:
+
+ - [Log into Bintray](https://bintray.com/login?forwardedFrom=%2Fkong%2F)
+ - Open the Profile settings (click the username at the top-right and select "Edit Profile")
+ - On the left select "API Key", and provide your password again
+
+To access the license try this:
+```bash
+BINTRAY_USERNAME="your_user_name@kong" && \
```

Comment:
Not sure why this includes `@kong`, but I think this would be confusing for users - I'd interpret this as "username"+"@kong". I'd recommend removing it.